src/App.js

```diff
@@ -1,24 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Routes, Route } from "react-router-dom";
 
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
We should import bootstrap in `index.js`, because it is an application-wide import.